students/Lincoln_Zhang/Session4/test_trigrams.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+import trigrams,os
+
+global_result = trigrams.read_file("sherlock_small.txt")
+
+def test_read_file():
+	assert global_result[0] == "One"
+	assert global_result[1] == "night"
+
+def test_build_trigram():
+	local_result = trigrams.build_trigram(global_result)
```

Comment:
here is where you can really isolate the tests. 

In this case, you want to test build_tirgram with a short, simple text, rather than ven the "small" real text, Perhaps:

"I wish I may I wish I might"

or a made up text that will test what you want. -- maybe one, two and three "followers"